tests/unit/utils/data.spec.js

```diff
@@ -189,12 +189,11 @@ describe('fetchDataForRouteEnter', () => {
   });
 
   it('calls `fetchData` with a configurable base url', async () => {
-    mockBaseUrl.mockReturnValueOnce('/base-prefix/');
```

Comment:
Why was this removed? It seems like it changes the meaning of this particular test.

Reply:
I fixed it mocking the `getAbsoluteUrl` function:
https://github.com/apple/swift-docc-render/pull/596/commits/7435a41d891fc28d7aecb48f0a7a5208d0850d0b

Reply:
couldnt we have used the real implementation?

Reply:
I couldn't make it work assigning `window.baseUrl` as you mentioned because now adding the baseUrl happens in the `normalizePath` function which it's a different scope, but if you have a suggestion of how to improve this test, please go ahead! :)